### PR TITLE
prep Dockerfiles for RH certification

### PR DESCRIPTION
**Pull request by sebgoa**
Same as I did for the sqs one but for 4 more that we can certify

### CODE REVIEW

File: awscognito/Dockerfile [hunk 1]
```diff
@@ -8,8 +8,19 @@ RUN go get github.com/golang/dep/cmd/dep && \
...
+      summary="The Triggermesh Cognito Source" \
+      description="This is the Triggermesh Knative Event Source for AWS Cognito"
+
+RUN microdnf update
+COPY ./licenses/* /licenses/
+
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 COPY --from=builder /go/bin/awscognito /go/bin/awscognito
 
-ENTRYPOINT ["/go/bin/awscognito"]
```
Comment (odacremolbap) [inline awscognito/Dockerfile]: typo maybe?

File: awscognito/Dockerfile [hunk 2]
```diff
@@ -8,8 +8,19 @@ RUN go get github.com/golang/dep/cmd/dep && \
...
+ENTRYPOINT ["/go/bin/sawscognito"]
```
Comment (odacremolbap) [inline awscognito/Dockerfile]:
```suggestion
ENTRYPOINT ["/go/bin/awscognito"]
```

Comment (odacremolbap) [reply awscognito/Dockerfile]: typo maybe

**Comment by sebgoa** [comment]
good catch, accepted suggestion.

I know we need tests :)

**Pull request merged**